src/hist/plot.py

```diff
@@ -231,14 +231,14 @@ def plot_pull(
     width = (right_edge - left_edge) / len(pulls)
     pull_ax.bar(self.axes.centers[0], pulls, width=width, **bar_kwargs)
 
-    patch_height = max(np.abs(pulls)) / pp_num
+    patch_height = 5. / pp_num
```

Comment:
So from the discussions this change is to be reverted back ...

Reply:
This could be an option?

Reply:
Yep. Maybe best to treat options in a new PR since there are many? So I would revert the 2 things suggested and we're good on this one.

Reply:
Agreed, focused PRs are best. :)